.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "submodules"]
-	path = submodules
-	url = https://github.com/ashishnegi/service-fabric-queryable
```

Comment:
>url [](start = 1, length = 3)

Dont we have dependency on this anymore?

Reply:
Yeah, we do have dependency. But the code in the referenced repo and actual Queryable that we use has diverged due to the fixes and all. I will shortly check in the queryable code in this repository too. Thus, this change.